src/engine-actions/create-wallet.js

```diff
@@ -3,25 +3,57 @@ const {
   initWallet
 } = require('../lnd-actions')
 
+/**
+ * Wallet recovery window number in blocks
+ * @constant
+ * @type {number}
+ * @default
+ */
+const RECOVERY_WINDOW_DEFAULT = 5000
```

Comment:
Why this number?

Reply:
Arbitrary default. This is a wallet address look-ahead. This recovery window will go back 5000 addresses that were generated from the seed. It seemed like a decent number for default until someone has an issue.